blank/README.md

```diff
@@ -4,21 +4,21 @@
 
 2. Install Go 1.4.x, git, setup `$GOPATH`, and `PATH=$PATH:$GOPATH/bin`
 
-3. Create PostgreSQL database.
+3. Get the source code.
```

Comment:
This README is for new developers wanting to install the project, not the person who initially ran go-bootstrap, right?